src/chunk.c

```diff
@@ -1486,6 +1608,9 @@ ts_chunk_create_for_point(const Hypertable *ht, const Point *p, bool *found, con
 			return chunk;
 		}
 
+		/* Lock compressed hypertable before resurrecting */
+		if (compress_ht)
```

Comment:
Using `compress_` to mean `compressed_` is slightly confusing, let's use `compressed_` everywhere?